libnetwork/controller.go

```diff
@@ -1299,3 +1299,23 @@ func (c *controller) iptablesEnabled() bool {
 	}
 	return enabled
 }
+
+func (c *controller) ip6tablesEnabled() bool {
```

Comment:
At first I was going to comment on the egregious layering violation. Then I saw the implementation of `(*controller).iptablesEnabled`  immediately above. Sigh, libnetwork is the 💩 that keeps on giving.

Reply:
`c.cfg.DriverCfg["bridge"]` is configuration for the `bridge` network driver. The controller is a layer above drivers and so should not have specific knowledge of how _any_ network driver is configured.

Reply:
Ah, didn't pay attention to the overall design when doing it. Now I see your point, thanks.